test/test_tree_1d_passive_tracers.jl

```diff
@@ -0,0 +1,35 @@
+module TestExamples1DEuler
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+EXAMPLES_DIR = pkgdir(Trixi, "examples", "tree_1d_dgsem")
```

Comment:
```suggestion
EXAMPLES_DIR = joinpath(examples_dir(), "tree_1d_dgsem")
```
Usually, we reuse the `examples_dir()`.